BouncyCastle-JCA/src/SSLContext.crysl

```diff
@@ -28,8 +28,8 @@ CONSTRAINTS
 	protocol in {"TLSv1.2"};
 
 REQUIRES
-	generatedKeyManager[kms];
-	generatedTrustManager[tms];
+	generatedKeyManagers[kms];
+	generatedTrustManagers[tms];
```

Comment:
Why di you add the 's'?

Reply:
Because there are two possible predicates that can be ensured in the respective [TrustManagerFactory](https://github.com/CROSSINGTUD/Crypto-API-Rules/blob/master/BouncyCastle-JCA/src/TrustManagerFactory.crysl#L32) and [KeyManagerFactory](https://github.com/CROSSINGTUD/Crypto-API-Rules/blob/master/BouncyCastle-JCA/src/KeyManagerFactory.crysl#L33) rules and the SSLContext rule requires the second predicate that both these rules can ensure.